erroranalysis/tests/test_cohort_filter.py

```diff
@@ -355,15 +392,47 @@ def create_iris_pandas():
     return X_train, X_test, y_train, y_test, feature_names
 
 
-def create_validation_data(X_test, y_test, pred_y=None):
+def create_validation_data(X_test, y_test, pred_y):
     validation_data = X_test.copy()
     validation_data[TRUE_Y] = y_test
     validation_data[ROW_INDEX] = np.arange(0, len(y_test))
-    if pred_y is not None:
-        validation_data[PRED_Y] = pred_y
+    validation_data[PRED_Y] = pred_y
     return validation_data
 
 
+def run_different_error_analyzers(validation_data,
```

Comment:
nit: maybe rename this to just
```run_error_analyzers```
I think this is shorter and more succinct, since "different" doesn't add more information to the method name